build.gradle

```diff
@@ -35,6 +35,11 @@ dependencies {
     compile 'org.springframework.boot:spring-boot-starter-security'
     compile 'org.springframework.boot:spring-boot-starter-aop'
     compile group: 'com.itextpdf', name: 'itextpdf', version: '5.5.13.1'
+    compile group: 'commons-fileupload', name: 'commons-fileupload', version: '1.4'
+    compile group: 'org.modelmapper', name: 'modelmapper', version: '2.3.0'
```

Comment:
Již je novější verze kdyžtak :)
compile group: 'org.modelmapper', name: 'modelmapper', version: '2.3.5'

Reply:
Tak tady budete mít konflikt ;)

Reply:
OK, dík, opravím to